src/odemis/driver/simcam.py

```diff
@@ -142,6 +143,48 @@ def __init__(self, name, role, image, dependencies=None, daemon=None, blur_facto
         # Convenience event for the user to connect and fire
         self.softwareTrigger = model.Event()
 
+        # show_message(None, 'starting and error wise thread', 'test error', timeout = 1.0)
+        self._errorCreationThread = threading.Thread(target = self._state_error_creation,
+                                                     args = (),
+                                                     name = "Creating and state error")
+        self._errorCreationThread.daemon = True
+        self._errorCreationThread.start()
+
+    def _state_error_creation(self):
```

Comment:
"creation" doesn't reality fit what it does, and the name doesn't indicate it's for a thread.
Maybe rename to `_state_error_run`.